src/page-loaders.ts

```diff
@@ -50,10 +50,10 @@ export const loadTagFeedAtomPage = async ({ tagId }: { tagId: string }) => {
 export interface WorkPage extends CheerioAPI {
   kind: "WorkPage";
 }
-export const loadWorkPage = async (workId: string) => {
+export const loadWorkPage = async (workId: string, chapterId?: string) => {
```

Comment:
Use the object params style as we do elsewhere, since it makes codes more readable and is more flexible when it comes to adding additional params:

```ts
export const loadWorkPage = async (params: { workId: string, chapterId?: string}) => {
```